x/upgrade/module.go

```diff
@@ -2,13 +2,13 @@ package upgrade
 
 import (
 	"encoding/json"
+	"github.com/cosmos/cosmos-sdk/client"
```

Comment:
gofmt is missing?

Reply:
yes, in couple of module.go files. Fixed now